frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,332 @@
+        }
+      }
+    }
+  }, [loading, csvName, namespace, resources]);
+
+  const getIndicator = () => {
+    if (isStatusFailed()) {
+      return <RedExclamationCircleIcon size="lg" />;
+    }
+    if (isApprovalNeeded()) {
+      return <YellowExclamationTriangleIcon size="lg" />;
+    }
+    if (isStatusSucceeded()) {
+      return <GreenCheckCircleIcon size="lg" />;
+    }
+    return <Spinner size="lg" />;
+  };
+
+  const getInstallMessage = () => {
+    if (isStatusFailed()) {
+      return (
+        <>
+          <span className="co-clusterserviceversion-install__heading">
+            Operator installation failed
+          </span>
+          <br />
+          <span>The operator did not install successfully.</span>
+        </>
+      );
+    }
+    if (isApprovalNeeded()) {
+      return (
+        <>
+          <span className="co-clusterserviceversion-install__heading">
+            Manual approval required
+          </span>
+          <br />
+          <p>
+            Review the manual install plan. Once approved, the following resources will be created
+            in order to satisfy the requirements for the components specified in the plan. Click the
+            resource name to view the resource in detail.
+          </p>
+        </>
+      );
+    }
+    if (isStatusSucceeded()) {
+      return (
+        <>
+          <span className="co-clusterserviceversion-install__heading">
+            Installed operator - ready for use
+          </span>
+        </>
+      );
+    }
+    return (
+      <>
+        <span className="co-clusterserviceversion-install__heading">Installing operator</span>
+        <br />
+        <span>The operator is being installed. This may take a few minutes.</span>
+      </>
+    );
+  };
+
+  const approve = () => {
+    k8sUpdate(InstallPlanModel, { ...installObj, spec: { ...installObj.spec, approved: true } })
+      .then(() => setStatus(''))
+      .catch((error) => {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      });
+  };
+
+  const getCSVLogo = () => {
+    if (typeof packageManifest === 'undefined') {
+      return <></>;
+    }
+    const channels = _.get(packageManifest, 'status.channels');
+    let chan: any = channels[0];
+    if ([...channels].length > 1) {
+      chan = [...channels].find((ch) => ch.currentCSV === csvName);
+    }
+    const displayName = _.get(chan, 'currentCSVDesc.displayName', '');
+    const logoVersion = _.get(chan, 'currentCSVDesc.version', '');
+
+    return (
+      <ClusterServiceVersionLogo
+        displayName={displayName}
+        icon={iconFor(packageManifest)}
+        provider={_.get(packageManifest, 'status.provider.name', '')}
+        version={logoVersion}
+      />
+    );
+  };
+
+  return (
+    <>
+      <Helmet>
+        <title>Operator Install</title>
+      </Helmet>
+      <Bullseye>
+        <div id="operator-install-page">
+          {loading && <div>Loading...</div>}
```

Comment:
I see this, its a little odd as a loading before installing messaging, it can make a strange flicker.

Reply:
This item is what I am wondering, will a card w/ a skeleton screen style treatment work.

Reply:
I updated the message to installing with a spinner.  We will have to follow up with a skeleton.